resources/views/includes/follow-button.blade.php

```diff
@@ -0,0 +1,21 @@
+@if($user->id == Auth::user()->id)
+    <a href="{{ route('settings') }}" class="btn btn-sm btn-primary">{{ __('profile.settings') }}</a>
+@elseif($user->private_profile && $user->followRequests->contains('user_id', Auth::user()->id))
+    <a href="#" class="btn btn-sm btn-primary disabled" aria-disabled="true">{{ __('profile.follow_req.pending') }}</a>
+@elseif($user->private_profile && !Auth::user()->follows->contains('id', $user->id))
+    <a href="#" class="btn btn-sm btn-primary follow" data-userid="{{ $user->id }}"
```

Comment:
```suggestion
    <a href="javascript:void(0)" class="btn btn-sm btn-primary follow" data-userid="{{ $user->id }}"
```

Reply:
But... why?

Reply:
They shouldn't. The whole JS has always `event.preventDefault();` in it, so `#` will not do anything. at least it doesn't for me.